src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -56,7 +56,7 @@ CREATE_LOGGERPTR_GLOBAL(logger_, "ProtocolHandler")
 std::string ConvertPacketDataToString(const uint8_t* data,
                                       const size_t data_size);
 
-const size_t kStackSize = 32768;
+const size_t kStackSize = 65536;
```

Comment:
I have no real problem with increasing the stack size, it just surprises me that a few extra calls would cause an overflow.